src/Hodor/Database/PgsqlAdapter.php

```diff
@@ -27,24 +27,85 @@ public function __construct(array $config)
         $this->config = $config;
     }
 
-    public function createJob($job)
+    /**
+     * @param string $queue_name
+     * @param array $job
+     */
+    public function bufferJob($queue_name, array $job)
     {
+        $row = [
+            'queue_name'    => $queue_name,
+            'job_name'      => $job['name'],
+            'job_params'    => json_encode($job['params']),
+            'buffered_at'   => $job['meta']['buffered_at'],
+            'buffered_from' => $job['meta']['buffered_from'],
+            'inserted_from' => gethostname(),
+        ];
+
+        if (isset($job['options']['run_after'])) {
+            $row['run_after'] = $job['options']['run_after'];
```

Comment:
Actually, scrap that. I think allowing for the `c`-formatted string like you're doing is fine. I'll add validation later when I add validation for other fields.